arelle/plugin/validate/DBA/ValidationPluginExtension.py

```diff
@@ -11,10 +11,12 @@
 
 _: TypeGetText
 
+DANISH_CURRENCY = 'DKK'
 NAMESPACE_CMN = 'http://xbrl.dcca.dk/cmn'
 NAMESPACE_FSA = 'http://xbrl.dcca.dk/fsa'
 NAMESPACE_GSD = 'http://xbrl.dcca.dk/gsd'
 NAMESPACE_SOB = 'http://xbrl.dcca.dk/sob'
+ROUNDING_MARGIN = 1000
```

Comment:
Out of curiosity, where does this value come from?

Reply:
This came from the validation spreadsheet: "There is an option to deviate by DKK 1000 in connection with any roundings"

Reply:
This looks like a constant rather than an option.